burger-app/src/components/OtherComponents/OrderDeteils/orderDeteils.js

```diff
@@ -0,0 +1,234 @@
+import styled from "styled-components";
+
+import { useState } from "react";
+
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+
+import { CircularProgress } from "@mui/material";
+
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import { validationRules } from "../../../utils/Validation/validations";
+import { SignupSchema } from "../../../utils/Validation/validationSchema";
+import { createOrder } from "../../../utils/Api/apiCall";
+import Image from "../../OtherComponents/Images/images";
+
+const OrderDetails = ({
+  isOpen,
+  orderPrice,
+  clearAll,
+  copyOrderIngredient,
+  handleClose,
+}) => {
+  const [loading, setLoading] = useState(false);
+  const [orderCreateStatus, setOrderCreateStatus] = useState(false);
+
+  const burgerIngredients = Object.entries(orderPrice).filter(
+    (product) => product[1] > 0
+  );
+
+  const handleOrder = async (data) => {
+    const {
+      name: orderName,
+      phone: orderPhone,
+      email: orderEmail,
+      address: orderAddress,
+    } = data;
+    const orderData = {
+      orderName,
+      orderPhone,
+      orderEmail,
+      orderAddress,
+      orderProducts: Object.fromEntries(burgerIngredients),
```

Comment:
It works not as it should 

<img width="414" alt="Screenshot 2023-01-12 at 17 10 55" src="https://user-images.githubusercontent.com/40695369/212105391-5c2e9c39-a6c2-4333-8f6b-bd7f78459d97.png">